build_tools/docker/context/install_iree_deps.sh

```diff
@@ -25,6 +25,11 @@ declare -a PACKAGES=(
     # Optional for tools like llvm-symbolizer, which we could build from
     # source but would rather just have available ahead of time
     llvm-dev
+    # Tracy build and run dependencies
```

Comment:
Packages here are installed for both base (18.04) and bleeding-edge (22.04). Tracy won't work with 18.04 due to the outdated version of libtbb.

We can guard this with an OS version check. But I don't really think that's necessary: 1. These packages are only a few MBs 2. 18.04 will be EoL in April 2023, I guess we will soon bump the minimum OS version.